presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -123,6 +123,7 @@
     public static final String OPTIMIZED_PARTITION_UPDATE_SERIALIZATION_ENABLED = "optimized_partition_update_serialization_enabled";
     public static final String PARTITION_LEASE_DURATION = "partition_lease_duration";
     public static final String CACHE_ENABLED = "cache_enabled";
+    public static final String MATERIALIZED_VIEW_MISSING_PARTITION_PREDICATE_COUNT_THRESHOLD = "materialized-view-missing-partition-predicate-count-threshold";
```

Comment:
The other properties are named using underline(`_`) instead of dash (`-`), we would want to keep it consistent.